### DISCUSSION TITLE
Unable to filter Date column with `datetime.date`

### DISCUSSION THREAD

**Discussion by ssnkhan** [saulpw/visidata#2112] [Q&A] [answered]
Hi folks,

I have searched this repo's issues and SO, and am still experiencing difficulties trying to filter rows based on date. I have a column, `access_date`, which I have cast as Date using `@`. This has worked, and the column displays a `@` beside the column name, and the cell value has been updated/truncated to `%Y-%m-%d`. I am now trying to filter these rows as follows:

1. Go to the `access_date` column
2. `z|`
3. `access_date < datetime.date(2023,01,01)` or `access_date < datetime.datetime(2023,01,01)`

In both cases above, Visidate returns 0 rows, when I can see values that would clearly match the filter expression. Would appreciate any help, thanks!

**Comment by ssnkhan** [comment]
Just tried `date(2023,01,01)` and that appears to have worked ... !

**Comment by aborruso** [comment]
For me it works using in example `data > datetime.date(2020,8,8)`

![image](https://github.com/saulpw/visidata/assets/30607/d28aef28-608c-4037-8c24-f615c2dc53ab)

**Reply by aborruso** [reply, marked as answer]
@ssnkhan I add that I have `import datetime` in my `~/.visidatarc` file

**Comment by ssnkhan** [comment]
You are a saint! Thank you so much!
